packages/extension/src/components/atoms/TruncatedMultiLinesText/TruncatedMultiLinesText.tsx

```diff
@@ -0,0 +1,32 @@
+import { ComponentProps } from 'react';
+
+import Typography from '@mui/material/Typography';
+
+type TruncatedMultiLinesTextProps = ComponentProps<typeof Typography> & {
+  text?: string;
+  maxLines?: string;
+};
+
+export const TruncatedMultiLinesText = ({
+  text = '',
+  maxLines = '4',
+  sx,
+  variant = 'body1'
```

Comment:
I forgot it yesterday when I sent the sandbox to you, I updated the sandbox now.

But you can replace this "variant" by `...rest` to take all the props from `Typography` and then you pass them to the `Typography` component below.

The full component:

```jsx
import Typography from "@mui/material/Typography";
import { ComponentProps } from "react";

type TruncatedMultiLinesTextProps = ComponentProps<typeof Typography> & {
  text?: string;
  maxLines?: string;
};

export const TruncatedMultiLinesText = ({
  text = "",
  maxLines = "4", // defaults to 4 max lines
  sx,
  ...rest
}: TruncatedMultiLinesTextProps) => {
  return (
    <Typography
      sx={{
        overflow: "hidden",
        display: "-webkit-box",
        "-webkit-box-orient": "vertical",
        "-webkit-line-clamp":
          maxLines /* start showing ellipsis when X line is reached */,
        "white-space": "pre-wrap" /* let the text wrap preserving spaces */,
        ...sx,
      }}
      {...rest}
    >
      {text}
    </Typography>
  );
};
```

Also `variant` is by default `body1`.

Reply:
Done, thank you!